src/compas_view2/app/app.py

```diff
@@ -113,15 +113,16 @@ def __init__(self, version='120', width=800, height=500, viewmode='shaded', cont
             app = QtWidgets.QApplication(sys.argv)
         app.references = set()
 
+        self.timer = None
         self.width = width
         self.height = height
         self.window = QtWidgets.QMainWindow()
         self.view = View(self, mode=viewmode)
         self.window.setCentralWidget(self.view)
         self.window.setContentsMargins(0, 0, 0, 0)
 
-        controller_cls = controller_cls or Controller
-        self.controller = controller_cls(self)
+        controller_class = controller_class or Controller
```

Comment:
should `controller_class` be `controllerclass` according to our discussion on naming convention?

Reply:
let's leave it for now. not super convinced anymore about my naming convention proposal :)

Reply:
cool : )